region/maridia/inner-green.json

```diff
@@ -1643,16 +2091,40 @@
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
-                    "canUseFrozenEnemies",
+                    "canTrickyUseFrozenEnemies",
                     {"or": [
-                      "Wave",
-                      "Spazer",
-                      "Plasma"
+                      "h_canCrouchJumpDownGrab",
+                      "canTwoTileSqueeze",
+                      "Morph"
                     ]}
                   ],
                   "note": [
                     "Involves breaking the top left Puyo free and then freezing it while it falls.",
-                    "It's very very tricky to get 4 shots off in time to freeze it. The beam requirements are to do it in less shots than that."
+                    "Walk to the end of the door platform and wait for the above puyo to land on the shot block.",
+                    "It will float there until its next jump, where it will slowly descend."
+                  ]
+                },
+                {
+                  "name": "Cross Room Gravity Jump",
+                  "notable": false,
+                  "requires": [
+                    "canTrickyJump",
+                    "canCrossRoomJumpIntoWater",
+                    "canMomentumConservingTurnaround",
+                    {"adjacentRunway": {
+                      "fromNode": 3,
+                      "usedTiles": 1,
+                      "physics": ["air"]
+                    }},
+                    {"or": [
+                      "canUseFrozenEnemies",
+                      "canSpringBallJumpMidAir",
+                      "HiJump"
+                    ]}
+                  ],
+                  "note": [
+                    "Quickly shoot to break the shot block and then do a momentumConservingTurnaround to ascend into the little region.",
```

Comment:
could use a devNote that this is unlikely to be possible because it relies on the next room being dry, and its likely going to be the pants room